app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,42 @@
+<resources>
+    <string name="app_name">Tv App Repo</string>
+    <string name="browse_title">Applis Tv</string>
+    <string name="related_movies">Vidéos similaires</string>
+    <string name="vertical_grid_title"><![CDATA[Grille vidéo verticale]]></string>
+    <string name="error">Erreur</string>
+    <string name="ok">OK</string>
+
+    <!-- Error messages -->
+    <string name="video_error_media_load_timeout">Le chargement du média ne répond pas</string>
+    <string name="video_error_server_inaccessible">Le serveur média n\'est pas joignable</string>
+    <string name="video_error_unknown_error">Erreur : impossible de charger la vidéo</string>
+    <string name="error_fragment_message">Une erreur s\'est produite</string>
+    <string name="dismiss_error">Annuler</string>
+    <string name="oops">Oops</string>
+    <string name="update">Mettre à jour</string>
+    <string name="uninstall">Désinstaller</string>
+    <string name="install">Installer</string>
+    <string name="app_not_leanback">Cette application n\'est pas optimisée pour les TV.</string>
+    <string name="app_leanback">Appli Leanback</string>
+    <string name="info_download_complete">Téléchargement terminé</string>
+    <string name="warning_invalid_tag">Tag non valide</string>
+    <string name="error_starting_intent">Erreur lors de l\'initialisation</string>
+    <string name="error_download_failed">Erreur : téléchargement échoué</string>
+    <string name="directory">Directory</string>
```

Comment:
Directory isn't used yet, but it will be a part of settings where users will be able to select where to download APKs, either to /downloads or to internal app memory.